src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1912,6 +1912,16 @@ static bool processSubscriptions
 
         if (cSubP != NULL)
         {
+          if (cSubP->status == STATUS_ONESHOT)
```

Comment:
What about CB working in `-noCache` mode? Have you tried the new .test with that configuration? It is a matter of just using the following line for broker start in the .test:

```
brokerStart CB 0 IPv4 -noCache
```

Reply:
Updated the implementation for noCache mode. 
Added the FT for noCache.
fixed in 8b354f2